Error in adding rectangle constraints in MATLAB

**Is your feature request related to a problem? Please describe.**
Consider a mobile robot is subject to state and input constraints. How can we set constraints on them? 

**Describe the solution you'd like**
I want to add constraints on input control signal (linear velocity and angular velocity) and states(x, y, theta). Could you please give an example in MATLAB?

**Describe alternatives you've considered**
I have looked in to the closed issue "Min/Max constraint for OpEnConstraints in Matlab #12". However, I'm struggling to add these in the example codes. 
I modified the main.rs file and added rectangle constraints on Matlab file `OpEnConstraints.m`, However, I'm always getting error as main.rs updates each time. This is what I'm getting:

```
error[E0425]: cannot find value `bounds` in this scope
  --> src/main.rs:82:37
   |
82 |         let problem = Problem::new(&bounds, df, f);
   |                                     ^^^^^^ not found in this scope

warning: unused import: `constraints::*`
 --> src/main.rs:3:27
  |
3 | use optimization_engine::{constraints::*, panoc::*, *};
  |                           ^^^^^^^^^^^^^^
  |
  = note: `#[warn(unused_imports)]` on by default
```

--- a/matlab/@OpEnConstraints/OpEnConstraints.m
+++ b/matlab/@OpEnConstraints/OpEnConstraints.m
@@ -44,6 +44,13 @@ classdef OpEnConstraints
             o.type = 'no_constraints';
             o.params = [];
         end
+
+        function o = make_rectangle(umin, umax)
+            o = OpEnConstraints();
+            o.type = 'rectangle';
+            o.params.umin = umin;
+            o.params.umax = umax;
+        end
     end
     
 end

--- a/matlab/@OpEnOptimizerBuilder/build.m
+++ b/matlab/@OpEnOptimizerBuilder/build.m
@@ -111,6 +111,16 @@ switch constraints.get_type()
         if ~(isfield(cstr_params, 'centre') && isempty(cstr_params.centre))
             fprintf(fid_main, '\n\t\tlet bounds = Ball2::new(None, %f);\n', cstr_params.radius);
         end
+    case 'rectangle'
+      
+        cstr_params = constraints.get_params();
+        umin = cstr_params.umin;
+        umax = cstr_params.umax;
+        s_min = sprintf('%.12f,', umin);
+        s_max = sprintf('%.12f,', umax);
+        fprintf(fid_main, '\n\t\tconst CONSTRAINTS_UMIN :Option<&[f64]> = Some(&[%s]); \n', s_min);
+        fprintf(fid_main, '\n\t\tconst CONSTRAINTS_UMAX :Option<&[f64]> = Some(&[%s]); \n', s_max);
+        fprintf(fid_main, '\n\t\tlet bounds = Rectangle::new(CONSTRAINTS_UMIN,CONSTRAINTS_UMAX);\n');
     case 'no_constraints'
         fprintf(fid_main, '\n\t\tlet bounds = NoConstraints::new();\n');
     otherwise

